Automattic/jetpack
# Form Block: hide CRM panel to non-admins.

Follow-up from #23618.

#### Changes proposed in this Pull Request:

* Since non-admins cannot manage plugins or access CRM plugin options, there is no use in showing them the CRM panel. Let's hide it.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start with a site connected to WordPress.com.
* Create a new user with an editor account.
* Log in with that user.
* Go to Posts > Add New and add a new form block.
* The CRM panel should not appear in the plugin sidebar (make sure you've selected the form block, not a child block).
* Log back in with the admin account.
* You should see the CRM panel then.
